devtools-ci-index.yaml

```diff
@@ -1889,6 +1898,25 @@
            allow-empty: true
            fingerprint: true
 
+- job-template:
+    name: '{ci_project}-{git_repo}-fabric8cd-publish'
+    description: |
+      {jobdescription}
+      If all goes well, publish artifacts to sonatype
+    node: "{ci_project}"
+    wrappers:
+        - vault-secrets:
+            <<: *vault_defaults
+            secrets:
+              - *npm-build-deliver-creds-gh-token
+              - *fabric8cd-sonatype-creds
+    properties:
+        - github:
+            url: https://github.com/{git_organization}/{git_repo}/
+        - inject:
+            properties-content: |
+               JENKINS_URL=https://ci.centos.org
+
```

Comment:
that `job-template` does not have any builders section and do not import any default object... so if you will define a job using this template that job will do nothing, is that what you wanted?